templates/admin/export.php

```diff
@@ -227,6 +228,7 @@
 <div class="export-control">
 	<p><input id="pb-export-button" type="button" class="button button-hero button-primary generate" value="<?php esc_attr_e( 'Export Your Book', 'pressbooks' ); ?>" /></p>
 	<p id="loader" class="loading-content"><span class="spinner"></span></p>
+	<?php if ( apply_filters( 'pb_export_show_files', true ) ) : ?>
```

Comment:
Expand the `<?php` and add

```
/**
 * @since 5.3.0
 *
 * TODO: Some explination as to why a user would use this
 *
 * @param bool $value
 */
```

Replace the TODO up there.